web/src/app/webview/build.sh

```diff
@@ -58,6 +63,9 @@ fi
 if builder_start_action build; then
   compile $SUBPROJECT_NAME
 
+  mkdir -p "$KEYMAN_ROOT/web/build/app/resources/osk"
+  cp -r "$KEYMAN_ROOT/web/src/resources/osk/" "$KEYMAN_ROOT/web/build/app/resources/"
```

Comment:
We need to ensure that GNU cp is the same as macOS/BSD cp here. (Once [bitten](https://github.com/keymanapp/keyman/pull/8565), twice shy.)